ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/ActionManager.java

```diff
@@ -93,5 +50,5 @@ private ActionManager()
 
 	private static final ActionManager _theManager = new ActionManager();
 
-	private static final Map<Uid, Lifetime> _allActions = new ConcurrentHashMap<Uid, Lifetime>();
```

Comment:
Thanks Manuel for your PR! Did you remove the static keyword (for _allActions map) on purpose? Could I ask you why?

Reply:
Because of the line above

Reply:
I see, thanks Manuel